libsrc/apple2/tgi/a2.lo.s

```diff
@@ -58,8 +58,8 @@ Y2      :=      ptr4
         .word   48              ; Y resolution
         .byte   16              ; Number of drawing colors
         .byte   1               ; Number of screens available
-        .byte   8               ; System font X size
-        .byte   8               ; System font Y size
+        .byte   0               ; System font X size
+        .byte   0               ; System font Y size
```

Comment:
As far as I can see the same is true for the C64 / C128 TGI drivers. Please change those too for the sake of consistency.

Reply:
No, it's just not implemented. I was thinking about integrating the Plato routine if the serial code ever works...

Reply:
> No, it's just not implemented.

I can't follow you. Isn't exactly this true for the C64 / C128 TGI drivers too?

> I was thinking about integrating the Plato routine if the serial code ever works...

Now you totally lost me. I thought PLATO is about 512x512 pixels. How  does that relate to this 40x48 pixel TGI ?

Reply:
Ah, now I see (?). For C64 / C128 TGI it's just a bug that they don't have text support, but for Apple II LoRes it's a hardware limitation.